src/rt/cover.c

```diff
@@ -715,6 +794,77 @@ void cover_count_tags(cover_tagging_t *tagging, int32_t *n_stmts,
    }
 }
 
+void cover_load_spec_file(cover_tagging_t *tagging, const char *path)
+{
+   cover_exclude_ctx_t ctx = {};
+
+   ctx.ef = fopen(path, "r");
+   if (ctx.ef == NULL)
+      fatal_errno("failed to open coverage specification file: %s\n", path);
+
+   tagging->spec = xcalloc(sizeof(cover_spec_t));
+
+   char *delim = " ";
```

Comment:
```suggestion
   const char *delim = " ";
```

Reply:
Is there actually some performance / code size benefit? Or is it just for clarity?

Reply:
It's a pointer to a read-only string: if the program writes through this it will segfault. Better to get a compile error instead.